src/osd/ReplicatedBackend.cc

```diff
@@ -778,58 +778,81 @@ int ReplicatedBackend::be_deep_scrub(
   const hobject_t &poid,
   ScrubMap &map,
   ScrubMapBuilder &pos,
-  ScrubMap::object &o)
+  ScrubMap::object& smap_object)
 {
   dout(10) << __func__ << " " << poid << " pos " << pos << dendl;
   auto& perf_logger = *(get_parent()->get_logger());
 
-  utime_t sleeptime;
-  sleeptime.set_from_double(cct->_conf->osd_debug_deep_scrub_sleep);
-  if (sleeptime != utime_t()) {
-    lgeneric_derr(cct) << __func__ << " sleeping for " << sleeptime << dendl;
-    sleeptime.sleep();
+  {
```

Comment:
I like extra scopes  too – it allows me to forget about e.g. `sleeptime` way earlier when I read the code.

Reply:
exactly why I added that. Easier for me to read